js/filteringBox.js

```diff
@@ -27,6 +27,52 @@ function openFilterModal() {
     closeFilterButton.classList.add("filter__box__topDiv__closeFilterBtn");
     closeFilterButton.innerHTML = "X";
```

Comment:
Using innerHTML can lead to potential vulnerabilities. Replacing it with .textContent or .createTextNode is a safer and more efficient approach. :)